app/commands/RemoveInactiveUsers.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Console;
+
+use App\Model\Entity\User;
+use App\Model\Repository\Users;
+use App\Helpers\AnonymizationHelper;
+use DateTime;
+use DateInterval;
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+/**
+ * A console command that removes inactive users.
+ * A user is inactive if no authentication (nor token renewal) is recorded for given period of time.
+ */
+class RemoveInactiveUsers extends Command {
+  /** @var string */
+  private $inactivityThreshold;
+
+  /** @var Users */
+  private $users;
+
+  /** @var AnonymizationHelper */
+  public $anonymizationHelper;
+
+  public function __construct(?string $inactivityThreshold, Users $users, AnonymizationHelper $anonymizationHelper) {
+    parent::__construct();
+    $this->inactivityThreshold = $inactivityThreshold;
+    $this->users = $users;
+    $this->anonymizationHelper = $anonymizationHelper;
+  }
+
+  protected function configure() {
+    $this->setName('users:remove-inactive')->setDescription('Remove users who has not been active for some time.');
+    $this->addOption('silent', null, InputOption::VALUE_NONE, 'Just do the job (no interactions, no reports)');
+  }
+
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    if (!$this->inactivityThreshold) return;
```

Comment:
PSR-2! :-)

Reply:
Yeah, we do not break lines at the beginning of function and class declarations (which is one of the best things in PSR, but you choose to ignore it), but let's badger poor ol' me for not spreading a simple guard over 3 lines.